mobly/controllers/android_device_lib/snippet_client.py

```diff
@@ -265,6 +265,11 @@ def _stop_event_client(self):
     """Releases all the resources acquired in `_start_event_client`."""
     if self._event_client:
       self._event_client.close_socket_connection()
+      # Without cleaning host_port of event_client, the event client will try to
+      # stop the port forwarding when deconstructed, which should only be
+      # stopped by the corresponding snippet client.
+      self._event_client.host_port = None
```

Comment:
should we release the port here before setting the var to None?

Reply:
The port forwarding should only be released by the corresponding snippet client. This is because that in `_start_event_client`, the event client directly uses the port of the snippet client without doing port forwarding.